packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard/signup-wizard.tsx

```diff
@@ -16,6 +17,7 @@ const SignupWizard = ({ closeWizard }: TSignupWizardProps) => {
     const [is_cancel_wizard_dialog_active, setIsCancelWizardDialogActive] = React.useState(false);
     const [current_step_key, setCurrentStepKey] = React.useState<string>();
     const [is_country_selected, setIsCountrySelected] = React.useState(false);
```

Comment:
@hamid-deriv I think this state is redundant, You can just check if `selected_country` has a value or not at line `69`.

Reply:
I'll check.

Reply:
Sorted.